GUI/GUIMod.cs

```diff
@@ -115,7 +115,27 @@ public GUIMod(CkanModule mod, IRegistryQuerier registry, KspVersionCriteria curr
             // KSP.
             if (latest_available_for_any_ksp != null)
             {
-                KSPCompatibility = KSPCompatibilityLong = registry.LatestCompatibleKSP(mod.identifier)?.ToString() ?? "any";
+                string str = registry.LatestCompatibleKSP(mod.identifier)?.ToString();
+                int major = registry.LatestCompatibleKSP(mod.identifier).Major;
+                int minor = registry.LatestCompatibleKSP(mod.identifier).Minor;
+                int patch = registry.LatestCompatibleKSP(mod.identifier).Patch;
+
+
+                if (str == null
+                    || (major + 1) % 10 == 0 
+                    || (major == 1 && (minor + 1) % 10 == 0)
```

Comment:
Treating 1.9 as "any" is going to get us in trouble if Squad ever releases a 1.10 or 2.0 version.